src/components/KuadrantOverviewPage.tsx

```diff
@@ -409,37 +411,41 @@ const KuadrantOverviewPage: React.FC = () => {
     return (
       <Popover
         className="custom-rounded-popover"
-        headerContent={`Error Code Distribution`}
+        headerContent={`Error Code`}
         bodyContent={
           <>
             <Text component={TextVariants.p}>
               {t('Displays the distribution of error codes for request failures.')}
             </Text>
-            {Object.entries(distribution).map(([code, dist]) => {
-              lastCode = code;
-              return (
-                <div key={code} style={{ marginBottom: '8px' }}>
-                  <Progress
-                    value={dist.percent}
-                    title={
-                      <Flex
-                        justifyContent={{ default: 'justifyContentSpaceBetween' }}
-                        alignItems={{ default: 'alignItemsCenter' }}
-                      >
-                        <FlexItem>
-                          <strong>Code: {code}</strong>
-                        </FlexItem>
-                        <FlexItem align={{ default: 'alignRight' }}>
-                          {dist.total.toFixed(0)} requests
-                        </FlexItem>
-                      </Flex>
-                    }
-                    measureLocation={ProgressMeasureLocation.outside}
-                  />
-                  <Divider style={{ margin: '12px 0' }} />
-                </div>
-              );
-            })}
+            <div className="custom-rounded-popover-codes">
```

Comment:
```suggestion
          <div style={{ maxHeight: '300px', overflowY: 'auto' }}>
```
I wasn't seeing the max height kicking in either had a quick look to see if i could figure it out. Found that adding a inline style over a custom CSS class works, why? no clue haven't looked to deep into it but it works.